src/db/dal/user.ts

```diff
@@ -1,24 +1,130 @@
+import AWS from 'aws-sdk';
 import createHttpError from 'http-errors';
 import { StatusCodes } from 'http-status-codes';
+import { Op, Order } from 'sequelize';
+import { uuid } from 'uuidv4';
 
+import { profileImageBucket } from '../../env';
 import { UserValidator } from '../../utils/userValidator';
 import UserModel, { IUserInput, IUserOuput } from '../models/User';
 
-const sanitizeInputPayload = (payload: IUserInput) => ({
-    ...payload,
-    id: undefined,
-    keycloak_id: undefined,
-    completed_registration: undefined,
-    creation_date: undefined,
-});
+let S3Client;
+try {
+    S3Client = new AWS.S3();
+} catch (error) {
+    //
+}
+
+const sanitizeInputPayload = (payload: IUserInput) => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { id, keycloak_id, completed_registration, creation_date, ...rest } = payload;
+    return rest;
+};
+
+//todo: use congif file for each project
+const otherKey = 'other';
```

Comment:
this name could imply a lot of things. What does that key do exactly?

Reply:
Ca vient d'include pour respecter l'existant. Mais comme j'ai mis en todo, je pense qu'il faut qu'on en fasse des configs au cas ou pour chaque project

Reply:
Ca mentionne le role "autre", la liste des roles est envoyé du front actuellement. Oui c'est pas terrible. On va changer ca dans une partie 2. D'abord le merge.